README_poolside.md

```diff
@@ -0,0 +1,20 @@
+# how to sync with upstream
+
+```
+git remote add upstream git@github.com:smacker/go-tree-sitter.git
+git fetch upstream
+git merge upstream/master # or some other branch that you want to merge with
+```
+
+There will be lots of conflicts. Unless we have changes we specifically made for poolside's use case and have not pushed to upstream, you probably will want to prefer upstream's version in any conflict.
+
+Then you need to fix up some C header #include directives, mostly because upstream is OK with having Unix shortcuts in them but Forge cannot build with such. There is a python script in this repo to help you with that:
+
+```
+# in the root of go-tree-sitter repo:
+./header_fix.py
+```
```

Comment:
Does it make sense to leverage [_autoamtion/main.go](https://github.com/smacker/go-tree-sitter/blob/master/_automation/main.go) for this purpose? It might be parametrised for a header prefix, or json config might be used.

Reply:
ye that's a good question.
i thought about it, but automatically generate a patch for that file is a bit harder, since it involved changing a few `replaceMap` into new downloadFiles. At that point we may as well just maintain a separate `main.go` + `grammer.json` fork. That's another approach